src/configurations/nf/resources.ts

```diff
@@ -2,10 +2,22 @@ export const datasetsSql = 'SELECT * FROM syn16859580'
 export const publicationsSql = 'SELECT * FROM syn16857542'
 export const studiesSql = 'SELECT * FROM syn16787123'
 export const initiativesSql = 'SELECT * FROM syn24189696'
-export const toolsSql = 'SELECT * FROM syn16859448'
+export const toolsSql = 'SELECT * FROM syn26438037'
 export const peopleSql = 'SELECT * FROM syn23564971'
-export const filesSql = `SELECT id AS "File ID", assay, dataType, diagnosis, tumorType,  species, individualID,  fileFormat, dataSubtype, nf1Genotype as "NF1 Genotype", nf2Genotype as "NF2 Genotype", studyName, fundingAgency, consortium, name AS "File Name", accessType, accessTeam  FROM syn16858331.9 WHERE resourceType = 'experimentalData'`
-export const metadataFilesSql = `SELECT id, dataType, assay, diagnosis, tumorType, species, individualID, fileFormat, dataSubtype, nf1Genotype, nf2Genotype, fundingAgency, consortium FROM syn16858331.9 where resourceType ='report'`
+export const filesSql = `SELECT id AS "File ID", assay, dataType, diagnosis, tumorType,  species, individualID,  fileFormat, dataSubtype, nf1Genotype as "NF1 Genotype", nf2Genotype as "NF2 Genotype", studyName, fundingAgency, consortium, name AS "File Name", accessType, accessTeam, Resource_id  FROM syn16858331.9 WHERE resourceType = 'experimentalData'`
+export const metadataFilesSql = `SELECT id, dataType, assay, diagnosis, tumorType, species, individualID, fileFormat, dataSubtype, nf1Genotype, nf2Genotype, fundingAgency, consortium FROM syn16858331.8 where resourceType ='report'`
 export const fundersSql = 'SELECT * FROM syn16858699'
 export const hackathonsSql = 'SELECT * FROM syn25585549'
-
+export const observationsSql = 'SELECT observationSubmitterName as "submitterName", synapseId as "submitterUserId", observationTime as "time", observationTimeUnits as "timeUnits", observationText as "text", observationType as "tag" FROM syn26470591 '
+export const investigatorSql = `SELECT investigatorName as "firstName", ' ' as "lastName", institution, investigatorSynapseId as "USERID" FROM syn26449831 WHERE (investigatorName IS NOT NULL OR investigatorSynapseId IS NOT NULL)`
```

Comment:
investigatorSql is not ideal.  The UserCardListRotate component expects the first name and last name to be in different fields, and this NF table only has the full name (investigatorName).  So I select investigatorName as the first name, and a constant ' ' for the last name.